components/Pamphlet/Pamphlet.jsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
You can use <> and don't have to import it - unless you need a key

Reply:
Oh dang I forgot about that shorthand, 👍.